public/index.html

```diff
@@ -115,7 +125,8 @@
                             <p class="heading">EKSAMINEERIJA</p>
                             <p class="content">PEEDO</p>
                         </div>
-                        <p class="description">Ronimisliit kinnitab, et ronijakaardi omanikul on piisavad osused iseseisvaks ronimiseks ja julgestamiseks kaljuronimisseinal ülaltjulgestuses.</p>
+                        <p class="description">Ronimisliit kinnitab, et ronijakaardi omanikul on piisavad osused
```

Comment:
Same 2 comments as above. Could those actually be merged together somehow and reused? Doesn't have to be in this PR.

Reply:
Possibly yeah, I don't like the duplication either. At some point the mobile and desktop designs seemed to diverge enough to warrant having separate markup, one with animations etc, but I'll have a look at some point if maybe it can be merged still.

Reply:
Yep, that makes sense!